templates/api/minimal_login.html

```diff
@@ -61,7 +61,6 @@
         <input type="submit" value="Login" />
         <input type="hidden" name="next" value="{{ next }}" />
     </form>
-    <p>Do not have a Freesound account? <a href="{% url "apiv2-registration" %}">Register here</a>.</p>
```

Comment:
I think it is still worth pointing to the normal registration page. I'll make that change

Reply:
oh yeah, I considered that. thanks